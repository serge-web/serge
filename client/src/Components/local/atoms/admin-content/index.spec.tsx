/* global it expect */
import React from 'react'
import renderer from 'react-test-renderer'
import AdminContent from './index'

it('content renders correctly', () => {
  const tree = renderer
    .create(<AdminContent>Text</AdminContent>)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
