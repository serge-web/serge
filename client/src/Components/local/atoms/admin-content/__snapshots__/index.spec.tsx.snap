// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`content renders correctly 1`] = `
<div
  className="main"
>
  Text
</div>
`;
